iris-mpc-common/src/helpers/statistics.rs

```diff
@@ -118,7 +118,18 @@ impl BucketStatistics {
 
             // The difference between buckets[i] and buckets[i - 1], except when i=0
             let previous_count = if i == 0 { 0 } else { buckets_array[i - 1] };
-            let count = buckets_array[i] - previous_count;
+            // Ensure non-decreasing cumulative counts to avoid underflow
+            let count = if buckets_array[i] >= previous_count {
+                buckets_array[i] - previous_count
+            } else {
+                tracing::warn!(
+                    "Non-monotonic cumulative bucket counts at index {}: current {}, previous {}; clamping to zero",
```

Comment:
I got randomly `"Panic occurred with message: attempt to subtract with overflow","backtrace" in iris-mpc-common/src/helpers/statistics.rs:121:25`

Added this guard here - but now cannot reproduce, so keeping it as warn so we can introspect later